biome.json

```diff
@@ -42,7 +42,6 @@
       },
       "complexity": {
         "noExtraBooleanCast": "error",
-        "noExtraSemicolon": "error",
```

Comment:
this emits error. Biome doesn't seem to have it.
and the rule seems to be removed from Rome as well

it doesn't exists when we click the link to [lint/complexity/noExtraSemicolon](https://docs.rome.tools/lint/rules/noExtraSemicolon)
from https://rome.tools/blog/2023/05/10/rome12_1/

Reply:
removed from Rome https://github.com/rome/tools/pull/4576